pkg/deepcopy/traverse.go

```diff
@@ -374,7 +374,8 @@ func (c *copyMethodMaker) genMapDeepCopy(actualName *namingInfo, mapType *types.
 				c.IfElse("val == nil", func() {
 					c.Line("(*out)[key] = nil")
 				}, func() {
-					c.Line("in, out := &val, &outVal")
+					c.Line("inVal := (*in)[key]")
+					c.Line("in, out := &inVal, &outVal")
```

Comment:
Thank you a lot for the contribution 🥇 
Could you please ensure that the change/issue is also covered with a test?

Reply:
Thanks for the review @camilamacedo86 I have added a nested map in cronjob_types.go and that fields and other fields are calling this new line. I hope that will cover as a test case. Can you suggest if I need to add test cases in other places as well?